source/_integrations/sensibo.markdown

```diff
@@ -45,127 +45,193 @@ then in the Sensibo app log you will be able to distinguish between actions
-
-## Button
-
-You can reset your filter check by using the button available on climate devices.
-
-By pressing the button, you tell your device that you have cleaned or replaced the filter.
+{% include integrations/config_flow.md %}
 
-## Number entities
+{% configuration_basic %}
+API key:
+  description: The previously created API key.
+{% endconfiguration_basic %}
 
-By using the number entities you can calibrate the temperature and hunmidity of your device.
+## Data fetching and limitations
 
-These entities are disabled by default.
+Data is polled from the **Sensibo** API once every minute for all devices.
 
-## Select entities
+If polling cannot happen because of no connectivity or a malfunctioning API, it will retry a few times before failing.
+The user can use the [`homeassistant.update_entity`](homeassistant#action-homeassistantupdate_entity) action to manually try again later, in the case the user has solved the connectivity issue.
 
-For supported devices, this integration provides support to set the following modes by the select entity:
+## Troubleshooting
 
-- Light
+This service is reliant on an internet connection and that the **Sensibo** API is available. Here are the things you can try before raising an issue:
 
-## Sensor entities
+- Check that internet is available from your Home Assistant instance.
+- Check that the **Sensibo** API is available by clicking [here](https://home.sensibo.com/api/v1/users/me). If you have previously logged in to Sensibo web, you will get a JSON back with the provided information about your account. If not logged in, the API will respond with `login_required`.
+- Use `curl` in a terminal on your Home Assistant instance using the same URL as previously opened in the browser. `curl https://home.sensibo.com/api/v1/users/me`
 
-For all devices, these sensors are available:
+### Specific log entries
 
-- Filter last reset
-- Feels Like
-- Timer end time
+**Log entry:** `Device [name of device] not correctly registered with remote on Sensibo cloud.`
 
-For motion sensors (supported by Sensibo Air devices), this integration provides the following sensors:
+When setting up a device the first time, a `remote` needs to be defined for the device in the **Sensibo** app either automatically or manually.
+The device will appear in Home Assistant, but won't be usable as no HVAC modes can be selected.
 
-- Temperature
-- Humidity
+## Entities
 
-For diagnostics, not automatically displayed on dashboards, these sensors are available for motion sensors:
+{% note %}
 
-- Voltage
-- Rssi
+Some entities are disabled by default, so you need to [enable them](/common-tasks/general/#to-enable-or-disable-a-single-entity) to use them.
 
-For Pure devices, these sensors are available:
+Depending on device support, some entities might not be available as the device does not support them.
 
-- PM2.5
-- Pure Boost Sensitivity
+{% endnote %}
 
-For AirQ device, these sensors are available:
+### Entities provided by all devices
```

Comment:
The Integration Quality Scale documents these differently. Please document the scale documentation for examples.